composer.json

```
@@ -37,19 +36,19 @@
         "lakion/mink-debug-extension": "^2.0.0",
         "phpspec/phpspec": "^7.0",
         "phpstan/extension-installer": "^1.0",
-        "phpstan/phpstan": "0.12.90",
-        "phpstan/phpstan-doctrine": "0.12.39",
-        "phpstan/phpstan-strict-rules": "^0.12.0",
-        "phpstan/phpstan-webmozart-assert": "0.12.12",
-        "phpunit/phpunit": "^9.5",
+        "phpstan/phpstan": "1.4",
```

```suggestion
        "phpstan/phpstan": "^1.4",
```
please use latest version